### PR TITLE
chore: rephrase package ownership statement + author link

### PR DESCRIPTION
**Pull request by iliapolo**
Screen Shot 2021-07-22 at 3 05 04 PM

### CODE REVIEW

File: src/views/Package/components/OperatorArea/OperatorArea.tsx [hunk 1]
```diff
@@ -39,12 +40,14 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
     }
 
     if (username) {
-      const managedBy = (
+      const managedBy = authorUrl ? (
```
Comment (RomainMuller) [inline src/views/Package/components/OperatorArea/OperatorArea.tsx:43]:
```suggestion
      const managedBy = author ? (
```

File: src/views/Package/components/OperatorArea/OperatorArea.tsx [hunk 2]
```diff
@@ -39,12 +40,14 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
...
+        <ExternalLink href={authorUrl}>{username}</ExternalLink>
+      ) : (
         <Box as="strong" fontWeight="bold">
           {username}
         </Box>
       );
-      items.push(<>Module managed by: {managedBy}</>);
+      items.push(<>Author: {managedBy}</>);
```
Comment (RomainMuller) [inline src/views/Package/components/OperatorArea/OperatorArea.tsx:50]:
```suggestion
      items.push(<>Author: {author}</>);
```

File: src/views/Package/components/OperatorArea/OperatorArea.tsx (hunk 1 above)
Comment (iliapolo) [reply src/views/Package/components/OperatorArea/OperatorArea.tsx:43]: Not sure I follow. What is `author`?

Comment (gabewomble) [reply src/views/Package/components/OperatorArea/OperatorArea.tsx:43]: Perhaps `const author = ...`, we still want to check if a URL exists to determine if we render a link or text

Comment (iliapolo) [reply src/views/Package/components/OperatorArea/OperatorArea.tsx]: Going to assume that yes thats what you meant - changed.

**Review by gabewomble — APPROVED** [review]

**Pull request merged**